src/node_buffer.h

```diff
@@ -19,9 +19,25 @@ NODE_EXTERN bool HasInstance(v8::Local<v8::Value> val);
 NODE_EXTERN bool HasInstance(v8::Local<v8::Object> val);
 NODE_EXTERN char* Data(v8::Local<v8::Value> val);
 NODE_EXTERN char* Data(v8::Local<v8::Object> val);
+
+#if ENABLE_TTD_NODE
+NODE_EXTERN void TTDAsyncModRegister(v8::Local<v8::Object> val,
+    byte* initialModPosition);
```

Comment:
> byte* initialModPosition); [](start = 3, length = 27)

Function parameters on new line should be on column after opening braces, see other APIs. Please fix all places.